language/stdlib/modules/SlidingNonce.move

```diff
@@ -21,6 +21,8 @@ module SlidingNonce {
     const ENONCE_TOO_NEW: u64 = 2;
     /// The nonce was already recorded previously
     const ENONCE_ALREADY_RECORDED: u64 = 3;
+    /// Nonce
```

Comment:
Please make a sentence out of this. This documentation is dumped by a debugging tool @tzakian wrote/is writing.